src/widget/loginscreen.cpp

```diff
@@ -40,7 +40,7 @@ LoginScreen::LoginScreen(QWidget *parent) :
     ui->setupUi(this);
 
     // permanently disables maximize button https://github.com/tux3/qTox/issues/1973
-    this->setWindowFlags(windowFlags() &! Qt::WindowMaximizeButtonHint);
+    this->setWindowFlags(windowFlags() &~ Qt::WindowMaximizeButtonHint);
```

Comment:
Well, no! Writing it a bit differently, it probably becomes more clear, that actually the `operator!` is used - which is very important here. Yet both the NOT and XOR operators relate to the flag and it probably has the same effect. :).

Just write it different:
```c++
this->setWindowFlags(windowFlags() & !Qt::WindowMaximizeButtonHint
```

All clear? :P